draft-ietf-scitt-architecture.md

```diff
@@ -683,15 +683,13 @@ Such policies may use as input all information in the Envelope, the Receipt, and
 
 # Privacy Considerations
 
-Transparency Services are often publicly accessible.
-Issuers should treat Signed Statements (rendering them as Transparent Statements) as publicly accessible.
-In particular, a Signed Statement Envelope and Statement payload should not carry any private information in plaintext.
+Transparency Services MAY support anonymous access.
+Issuers MUST ensure Signed Statements submitted to public access services are acceptable for public disclosure.
```

Comment:
```suggestion
Issuers SHOULD ensure Signed Statements submitted to public access services are acceptable for public disclosure.
```

Reply:
Option 2, changing should to SHOULD for the original text
```suggestion
Issuers SHOULD treat Signed Statements (rendering them as Transparent Statements) as publicly accessible.
```

Reply:
I liked OPTION 1 from Steve, because only Transparency Service with  public access is allowed public disclosure.

There may be a specific deployment where only designated actors may have access, with one the designated actor is an Auditor. There the Privacy considerations may vastly differ from others!